ui/pages/settings/experimental-tab/experimental-tab.component.js

```diff
@@ -261,7 +261,17 @@ export default class ExperimentalTab extends PureComponent {
               id="select-theme"
               options={themesOptions}
               selectedOption={theme}
-              onChange={async (newTheme) => setTheme(newTheme)}
+              onChange={(newTheme) => {
+                this.context.trackEvent({
+                  category: 'Settings',
+                  event: 'Theme Change',
```

Comment:
[req] similar to "Token Added" and "Clicked ...${x}" lets make this at least "Changed"